mbrl/models/gaussian_mlp.py

```diff
@@ -77,19 +79,19 @@ def __init__(
         num_layers: int = 4,
         ensemble_size: int = 1,
         hid_size: int = 200,
-        use_silu: bool = False,
         deterministic: bool = False,
         propagation_method: Optional[str] = None,
         learn_logvar_bounds: bool = False,
+        activation_cls = None,
     ):
         super().__init__(
             ensemble_size, device, propagation_method, deterministic=deterministic
         )
 
         self.in_size = in_size
         self.out_size = out_size
-
-        activation_cls = nn.SiLU if use_silu else nn.ReLU
+        if activation_cls is None:
+            activation_cls = nn.ReLU
```

Comment:
One issue with this is that, as far as I know, it won't work with `hydra` (see [config](https://github.com/facebookresearch/mbrl-lib/blob/master/mbrl/examples/conf/dynamics_model/gaussian_mlp.yaml)). If we want to pass the class directly, we should do something like [this config](https://github.com/facebookresearch/mbrl-lib/blob/master/mbrl/examples/conf/dynamics_model/basic_ensemble.yaml#L9), and instantiate like [this](https://github.com/facebookresearch/mbrl-lib/blob/fa3f9b922db9b61d81cafb359546c5bb78440cd6/mbrl/models/basic_ensemble.py#L74) in the code.

Reply:
Sorry, the above was probably confusing. What I meant is that we cannot pass custom objects directly due to `Hydra`. I left other follow up comments about using a config for the activation.